### PR TITLE
Kafka develop2 v3

### PR DESCRIPTION
**Pull request by sowmya695**

### CODE REVIEW

File: registration-processor-default.properties [hunk 1]
```diff
@@ -999,3 +999,5 @@ mosip.registration.processor.print.template=RPR_UIN_CARD_TEMPLATE
 
 # this property is used to define issuers eg partnerid:credentialType:templateTypeCode
 mosip.registration.processor.issuer=mpartner-default-print:${mosip.registration.processor.print.credentialtype}:${mosip.registration.processor.print.template};mpartner-default-digitalcard:${mosip.registration.processor.digitalcard.credentialtype}:${mosip.registration.processor.digitalcard.template}
+##timeout in milliseconds for health check registrer
+mosip.regproc.health-check.handler-timeout=2000
```
Comment (vishwa-vyom) [inline registration-processor-default.properties:1003]: why 5 seconds as default and 2 seconds in the config ?

Comment (vishwa-vyom) [reply registration-processor-default.properties:1003]: Can we keep the proper default in the code and avoid adding it in the config ?

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: registration-processor-default.properties (hunk 1 above)
Comment (sowmya695) [reply registration-processor-default.properties:1003]: Fixed

**Comment by sowmya695** [comment]
Default value is present in code so cancelling the review

**Pull request closed without merging**